gui/src/installer/mod.rs

```diff
@@ -33,6 +33,7 @@ pub struct Installer {
     steps: Vec<Box<dyn Step>>,
     hws: HardwareWallets,
     signer: Arc<Mutex<Signer>>,
+    back_button: bool,
```

Comment:
Is the idea that if the user passes the network in the command line, then it would not be possible to go back? Perhaps in that case, this field could be changed to `network: Option<Network>` so that it stores this pre-selected network, if any, and we could also use it on the `DefineDescriptor` step to select this same network.

Reply:
i was not aware we can pass network as cli arg, i need to look at this,

Reply:
just look at this, to me it look that if the user select a network from cli, why would it go back to choose an other network?

The logic actually implemented is:
- if Installer is launch at startup, do not display the `Previous` button ( we land there in 2 cases: no wallet configured yet or network pass by cli)
- if we land to Installer from the Launcher, display the `Previous` button

i can change the logic if you think we should alway can go back to Launcher

Reply:
That's OK, I think it makes sense not to show the Previous button in those cases.